prefect_kubernetes/utilities.py

```diff
@@ -0,0 +1,70 @@
+""" Utilities for working with the Python Kubernetes API. """
+from pathlib import Path
+from typing import TypeVar, Union
+
+from kubernetes.client import models as k8s_models
+from prefect.infrastructure.kubernetes import KubernetesJob, KubernetesManifest
+
+base_types = {"str", "int", "float", "bool", "list[str]", "dict(str, str)", "object"}
+
+V1KubernetesModel = TypeVar("V1KubernetesModel")
+
+
+def convert_manifest_to_model(
```

Comment:
Maybe we can use the utils:

```
from kubernetes import client, config, utils
config.load_kube_config()
k8s_client = client.ApiClient()
yaml_file = '<location to your multi-resource file>'
utils.create_from_yaml(k8s_client, yaml_file)
```

https://stackoverflow.com/questions/56673919/kubernetes-python-api-client-execute-full-yaml-file

There's also: `utils.create_from_dict()`

Reply:
Their utils are very similar, but they actually create the resource when you call `create_from_{manifest_type}`. 

I like the idea of being able to create / pass around the python object without being forced to actually create an instance of that resource against your cluster, but not sure, I could be convinced to switch to their utils.